contrib/runners/announcement_runner/announcement_runner/announcement_runner.py

```diff
@@ -60,7 +60,13 @@ def run(self, action_parameters):
         self._dispatcher.dispatch(self._route,
                                   payload=action_parameters,
                                   trace_context=trace_context)
-        return (LIVEACTION_STATUS_SUCCEEDED, action_parameters, None)
+
+        result = {
+            "output": action_parameters
+        }
+        result.update(action_parameters)
+
+        return (LIVEACTION_STATUS_SUCCEEDED, result, None)
```

Comment:
I see, so the output format changes here - will we add upgrade notes entry for it?

Reply:
Yes, but it should also be backwards compatible so long as the user isn't iterating over all keys.